packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -7,6 +7,40 @@
       "accessDeniedHelp": "",
       "imprint": "",
       "privacy": ""
+    },
+    "roles": {
```

Comment:
Just thinking a bit about the naming, maybe `shareRoles` would be more fitting 🤔 What do you think?